examples/flutter_gallery/lib/demo/contacts_demo.dart

```diff
@@ -117,15 +120,16 @@ class ContactsDemoState extends State<ContactsDemo> {
             SliverAppBar(
               expandedHeight: _appBarHeight,
               pinned: _appBarBehavior == AppBarBehavior.pinned,
-              floating: _appBarBehavior == AppBarBehavior.floating || _appBarBehavior == AppBarBehavior.snapping,
+              floating:
+                  _appBarBehavior == AppBarBehavior.floating || _appBarBehavior == AppBarBehavior.snapping,
```

Comment:
not a big fan of this, but i think this speaks to the "hard vs soft line length" things i've written about in other contexts